meta-iotqa/lib/oeqa/runtime/sanity/pulseaudio.py

```diff
@@ -0,0 +1,12 @@
+from oeqa.oetest import oeRuntimeTest
+from oeqa.utils.decorators import tag
+
+class PulseaudioDaemonTest(oeRuntimeTest):
+
+    '''Pulseaudio daemon check'''
+    def test_pulseaudio_daemoncheck(self):
+        '''check pulseaudio daemon'''
```

Comment:
Maybe rename the the class as 'PulseaudioTest' and the test function as 'test_pulseaudio_exists' and also the comments to reflect those.